dotcom-rendering/index.d.ts

```diff
@@ -551,6 +551,25 @@ interface TagType {
 	bylineImageUrl?: string;
 }
 
+/**
+ * BlocksRequest is the expected body format for POST requests made to /Blocks
+ */
+interface BlocksRequest {
+	blocks: Block[];
+	format: CAPIFormat;
+	host?: string;
+	pageId: string;
+	webTitle: string;
+	ajaxUrl: string;
+	isAdFreeUser: boolean;
+	isSensitive: boolean;
+	edition: string;
+	section: string;
+	sharedAdTargeting: Record<string, unknown>;
+	adUnit: string;
+	videoDuration?: number;
```

Comment:
Small but I tend to order these alphabetically. VS Code has a way to “sort lines ascending”.

Reply:
https://eslint.org/docs/rules/sort-keys

Reply:
That `sort-keys` could make sense, but when combined with TS, and some of the points @oliverlloyd makes (required/optional), with a few other points (shorthand vs explicit `{ a, c, b: thingy }`) - you could get some pretty strange looking code (strange being subjective).

Good way to try is to add the linting rule, auto lint and look at the diff?